src/Components/InventoryPage/__tests__/InventoryPage.test.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+
+import InventoryPage from '../InventoryPage';
+import { useGetEntities } from '../hooks';
+import { findCheckedValue } from '../helpers';
+
+const systemsMock = {
+  total: 2,
+  count: 2,
+  page: 1,
+  per_page: 10,
+  results: [
+    {
+      display_name: 'rhiqe.8d8fe668-c2ee-4034-b174-2af31020402c.email-38.alvarez.com',
+      id: '1d2a2d8b-8bb1-48f3-a367-c8ba2f4d8fc3',
+      system_profile: {
+        operating_system: {
+          major: 8,
+          minor: 6,
+          name: 'RHEL',
+        },
+      },
+      selected: true,
+      updated: '2022-04-20T12:49:29.389485Z',
+    },
+    {
+      display_name: 'rhiqe.laptop-44.sanchez-mann.biz',
+      id: 'aa28f81c-e7f0-4ee4-af46-0352b0fb50f3',
+      system_profile: {
+        operating_system: {
+          major: 8,
+          minor: 8,
+          name: 'RHEL',
+        },
+      },
+      selected: false,
+      updated: '2022-05-04T07:08:37.176240Z',
+    },
+  ],
+};
+
+const systemTags = {
+  total: 2,
+  count: 2,
+  page: 1,
+  per_page: 10,
+  results: {
+    '1d2a2d8b-8bb1-48f3-a367-c8ba2f4d8fc3': [
+      {
+        namespace: 'insights-client',
+        key: 'application',
+        value: 'TEST',
+      },
+      {
+        namespace: 'insights-client',
+        key: 'cloud',
+        value: 'POG',
+      },
+    ],
+  },
+};
+
+jest.mock('../../../api', () => {
+  return {
+    fetchSystems: jest.fn(() => Promise.resolve(systemsMock)),
+    fetchSystemsTags: jest.fn(() => Promise.resolve(systemTags)),
+  };
+});
+
+jest.mock('../../../Helpers/Hooks', () => {
+  return {
+    useRbac: () => [[true], false],
+  };
+});
+
+describe('InventoryTable', () => {
+  let mockStore = configureStore();
+  let props;
+  const setSelectedIds = jest.fn();
+
+  it('should render correctly', () => {
```

Comment:
Is it possible to test somewhere that correct content renders if the permissions are not sufficient? Maybe I just don't see it :thinking:

Reply:
Yes we can, but we need to make mockup useRbac for individual test cases, but I didn't find out how to do it. But I will try :+1:

Reply:
Done.